configure.ac

```diff
@@ -0,0 +1,77 @@
+dnl Process this file with autoconf to produce a configure script.
+
+AC_PREREQ(2.61)
+
+AC_INIT([mate-submodules], [1.24.0], [https://mate-desktop.org])
+AM_INIT_AUTOMAKE([1.9 foreign dist-xz no-dist-gzip check-news])
+m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
+
+AC_CONFIG_SRCDIR([config.h.in])
+AC_CONFIG_HEADER([config.h])
+AC_CONFIG_MACRO_DIR([m4])
+
+MATE_COMMON_INIT
+MATE_DEBUG_CHECK([no])
+MATE_COMPILE_WARNINGS
+
+AC_PROG_CC
+AM_DISABLE_STATIC
+AC_PROG_LIBTOOL
+PKG_PROG_PKG_CONFIG
+GLIB_GSETTINGS
```

Comment:
GLIB_GSETTINGS ? Why do you add this macro if there is no GSettings schema XML file?